Packaging/OpenDingux/buildroot_rg350_defconfig

```diff
@@ -0,0 +1,20 @@
+BR2_mipsel=y
+BR2_mips_32r2=y
+# BR2_MIPS_SOFT_FLOAT is not set
+BR2_OPTIMIZE_2=y
+BR2_KERNEL_HEADERS_3_12=y
+BR2_TOOLCHAIN_BUILDROOT_LARGEFILE=y
+BR2_TOOLCHAIN_BUILDROOT_INET_IPV6=y
+BR2_TOOLCHAIN_BUILDROOT_WCHAR=y
+BR2_BINUTILS_EXTRA_CONFIG_OPTIONS="--enable-lto"
+BR2_GCC_VERSION_4_9_X=y
+BR2_EXTRA_GCC_CONFIG_OPTIONS="--enable-lto"
+BR2_TOOLCHAIN_BUILDROOT_CXX=y
+BR2_PACKAGE_SDL2_IMAGE=y
```

Comment:
I don't know exactly what this does, but SDL2_image isn't required by the project.

Reply:
It is required by the buildroot, they currently compile SDL2 expecting IMAGE to be there.

Reply:
We could try to customize it but I think the effort is just not worth it at this point.